packages/convex-helpers/package.json

```diff
@@ -30,11 +79,12 @@
     "validators.ts"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "node generate-exports.mjs && tsc",
     "prebuild": "npm run test",
     "test": "tsc --project tsconfig.test.json",
     "clean": "rm -rf dist",
-    "watch": "chokidar '*.ts' 'server/*.ts' 'react/*.ts' 'tsconfig.json' 'package.json' -c 'npm run build' --initial"
+    "watch": "chokidar '*.ts' 'server/*.ts' 'react/*.ts' 'tsconfig*.json' 'package.json' -c 'npm run build' --initial",
+    "arethetypeswrong": "npx attw $(npm pack)"
```

Comment:
presumably this should be called by the publish script?

Reply:
it's a lint for troubleshooting, we could run it with tests